app/liquid_tags/blogcast_tag.rb

```diff
@@ -0,0 +1,41 @@
+class BlogcastTag < LiquidTagBase
+  def initialize(tag_name, id, tokens)
+    super
+    @id = parse_id(id)
+  end
+
+  def render(_context)
+    html = <<-HTML
+      <div class="ltag_blogcast"
+        style="position: relative;
+          width: 100%;
+          height: 0;">
+        <iframe frameborder="0"
+          scrolling="no"
+          id="blogcast_#{@id}"
+          mozallowfullscreen="true"
+          src="//blogcast.host/embed/#{@id}"
+          style="width:100%;height:132px;overflow:hidden;"
+          webkitallowfullscreen="true"></iframe>
+      </div>
+    HTML
+    finalize_html(html)
+  end
+
+  private
+
+  def parse_id(input)
+    input_no_space = input.delete(" ")
+    if valid_id?(input_no_space)
+      input_no_space
+    else
+      raise StandardError, "Invalid Blogcast Id"
+    end
+  end
+
+  def valid_id?(id)
+    id =~ /\d{1,9}/i
```

Comment:
The current regex allows XSS to get through. The following suggestion should be strict enough to prevent it.
```suggestion
    (id =~ /\A\d{1,9}\Z/i)&.zero?
```

The XSS example I used: `{% blogcast 123131\"onload="alert()" %}`